### PR TITLE
docs: Update ja-jp.xaml

### PR DESCRIPTION
**Pull request by wallsman**
mistranslation fixed(誤訳修正) / Japanese tweaked(わかりにくい部分を修正) / Timer line Combined with English translation（タイマー部分がかなり分かりづらかったので英語訳と合わせた）/ Add comment out(翻訳修正候補など追加)

### CODE REVIEW

**Review by Tastror — APPROVED** [review]

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml [hunk 1]
```diff
@@ -179,28 +179,28 @@
...
+    <system:String x:Key="LinkStart">Link Start!</system:String><!--分かりやすく動作開始でも良い気はする-->
     <system:String x:Key="Stop">止める</system:String>
 
     <!--  戦闘設定  -->
-    <system:String x:Key="UseSanityPotion">回復剤使う</system:String>
-    <system:String x:Key="UseOriginitePrime">源石使う*</system:String>
-    <system:String x:Key="PerformBattles">回数限定*</system:String>
-    <system:String x:Key="AssignedMaterial">素材限定</system:String>
+    <system:String x:Key="UseSanityPotion">理性回復剤の使用数</system:String><!--UIの表示バランス大丈夫？-->
+    <system:String x:Key="UseOriginitePrime">純正源石の使用数*</system:String><!--UIの表示バランス大丈夫？-->
```
[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:187]: To put it in layman's terms.
”理性剤使用数”
”割る石の数”
I think that would also work.

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml [hunk 2]
```diff
@@ -179,28 +179,28 @@
...
-    <system:String x:Key="StageSelect2">代わり</system:String>
-    <system:String x:Key="Amend">改定</system:String>
+    <system:String x:Key="StageSelect2">実行できない際の代替</system:String>
+    <system:String x:Key="Amend">設定の変更</system:String>
 
     <!--<system:String x:Key="CurrentStage">現在ステージ</system:String>
     <system:String x:Key="LastBattle">前回ステージ</system:String>-->
     <system:String x:Key="DefaultStage">現在/前回</system:String>
-    <system:String x:Key="ClosedStage">イベントが終わった</system:String>
+    <system:String x:Key="ClosedStage">終了イベント</system:String><!--動作不明-->
```
[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:198]: I left the comment out because I could not read the words and the English.
The words could have been "side stories" or "episodes".

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml [hunk 3]
```diff
@@ -179,28 +179,28 @@
...
-    <system:String x:Key="AssignedMaterial">素材限定</system:String>
+    <system:String x:Key="UseSanityPotion">理性回復剤の使用数</system:String><!--UIの表示バランス大丈夫？-->
+    <system:String x:Key="UseOriginitePrime">純正源石の使用数*</system:String><!--UIの表示バランス大丈夫？-->
+    <system:String x:Key="PerformBattles">周回数指定*</system:String>
+    <system:String x:Key="AssignedMaterial">素材を限定</system:String>
     <system:String x:Key="Quantity">ドロップ数</system:String>
     <system:String x:Key="StageSelect">ステージ</system:String>
-    <system:String x:Key="StageSelect2">代わり</system:String>
-    <system:String x:Key="Amend">改定</system:String>
+    <system:String x:Key="StageSelect2">実行できない際の代替</system:String>
```
[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: If you use only "代替" in Japanese, it is not clear what is being substituted, so it would be better to annotate with a tip.

[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: If you use only "代替" in Japanese, it is not clear what is being substituted, so it would be better to annotate with a tip.

[the remark this answers is not in the document]
Comment (Tastror) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: what about 「候補」

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (zzyyyl) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:198]: It just means "activity ended".
maybe `活動終了`

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tastror) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:187]: It sounds good

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:198]: I think...
> ステージ終了
> タスク終了

If you want to write it straight...
"イベント終了"

or something like that.

**Comment by MistEO** [comment]
For code changes, please commit to the `dev` branch, thanks

File: src/MeoAsstGui/Resources/Localizations/ja-jp.xaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: Same.
I think that if you don't annotate with hints, etc., the operation will not be understood in Japanese.

A candidate would be smarter to add a display of annotations of the contents of the action to the empty space at the bottom of the UI.

Or add an addendum to the manual, "Please read the manual." to guide the user.

[the remark this answers is not in the document]
Comment (Tastror) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: Yes.
But in Chinese client we use 备选(Candidate), which is similar to the expression in Japanese client.
Since users will see this word only if they have checked this option in settings by their own, we don't need to worry about their misunderstanding.

[the remark this answers is not in the document]
Comment (zzyyyl) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: what about 「代替ステージ」

[the remark this answers is not in the document]
Comment (wallsman) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: If the above is correct, there should be no problem.

[the remark this answers is not in the document]
Comment (Tastror) [reply src/MeoAsstGui/Resources/Localizations/ja-jp.xaml:192]: ～

**Comment by zzyyyl** [comment]
> For code changes, please commit to the `dev` branch, thanks

@wallsman This patch should be merged into dev, so you should rebase this branch to dev~

**Comment by wallsman** [comment]
Ah, i forget it...
Sorry guys.

**Comment by Tastror** [comment]
```
    <system:String x:Key="CustomStageCode">ステージ名を入力する</system:String>
    <system:String x:Key="CustomStageCodeTip">ステージの原名と番号（4-10、6-16など）の両方がサポートされています。</system:String>
```

The dev branch has added these two new sentences, you can comment if they need to be modified.

1, type in the name of the stage  
2, We support both the full name and the short name of the stage.

@wallsman

**Comment by Tastror** [comment]
> @Tastror Why did you merge dev into this branch, commit [Update ja-jp.xaml](https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/1775/commits/4feadfbe0b20a397a1eb45285818c0662baa14e3) is still on top master and to be picked onto dev 🤔

I just try to resolve the conflict on GIthub...

**Comment by horror-proton** [comment]
Now dev is at https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/de31a644b197b467eedf0b5686af91405cc3f327 now, perhaps I should fast forward it to master on https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/71fa8e0cda3543171d54b2053bbdd32612d7d3e6 now, (when it's not too late)

**Comment by Tastror** [comment]
I'd like to merge this pull request to `dev` now, and don't need to worry about the dev head since`dev` will merge to `master` soon or later.
If something need to be altered, we can create another PR based on `dev`.

**Comment by wallsman** [comment]
> The dev branch has added these two new sentences, you can comment if they need to be modified.
> 
> 1, type in the name of the stage
> 2, We support both the full name and the short name of the stage.

>  <system:String x:Key="CustomStageCodeTip">ステージの原名と番号（4-10、6-16など）の両方がサポートされています。</system:String>

`<system:String x:Key="CustomStageCodeTip">ステージ名(例:暴君など)と、番号（例:4-10、6-16など）の両方がサポートされています。</system:String>`

If the stage name is to be recognized by OCR, the Japanese name is fine, but if it cannot be recognized because the roguelike operator specified a Chinese name, I think it would be better to have an annotation stating that "Chinese name input is required.
